__manifest__.py

```diff
@@ -0,0 +1,25 @@
+{
+    'name': 'Purchase Order Enhancement',
+    'version': '1.0',
+    'license': 'LGPL-3',
+    'depends': [
+        'base',
+        'purchase'
+    ],
+    'data': [
+        'data/ir_cron_archive_po.xml',
+        'report/purchase_order_pdf.xml',
+        'report/purchase_order_analysis_pivot.xml',
+        'security/ir.model.access.csv',
+        'views/purchase_order_enhancement.xml',
+        'wizard/purchase_order_archives_view.xml',
+    ],
+    'assets': {
+        'web.assets_backend': [
+            'purchase_order_enhancement/static/src/js/us_phone.js',
+        ],
+    },
+    'installable': True,
+    'application': True,
```

Comment:
This module contains customizations for the purchase module of Odoo. This module itself is not an application. So please mark `application` as False here